NitroxPatcher/Patches/Persistent/uGUI_MainMenu_Start_Patch.cs

```diff
@@ -0,0 +1,41 @@
+#if DEBUG
+using System;
+using System.Reflection;
+using HarmonyLib;
+using NitroxClient.MonoBehaviours.Gui.MainMenu;
+using NitroxModel.Helper;
+
+namespace NitroxPatcher.Patches.Persistent;
+
+public sealed partial class uGUI_MainMenu_Start_Patch : NitroxPatch, IPersistentPatch
```

Comment:
Should we keep it ?

Reply:
I'd rather keep it before we get launcher merged (will not be instant I'm sure of it).
There'll be a considerable meantime durinh which it'll be cool to have people use the tool

Reply:
I agree. It's not a big deal to resolve this merge conflict later.